templates/cf-execd.service.mustache

```diff
@@ -2,11 +2,12 @@
 Description=CFEngine Execution Scheduler
 After=syslog.target
 ConditionPathExists={{{vars.sys.bindir}}}/cf-execd
+ConditionPathExists={{{vars.sys.workdir}}}/inputs/promises.cf
```

Comment:
Don't run execd unless there is a default policy entry.

Instead of `vars.sys.workdir`, I would probably suggest `vars.sys.inputdir` or even better  `vars.sys.default_policy_path` (available since 3.13.0), so I guess probably not yet backwards compatible enough.

Reply:
Yep. Can adjust all the various vars to something more precise. :+1: